pallets/subtensor/src/macros/dispatches.rs

```diff
@@ -2309,5 +2311,120 @@ mod dispatches {
             ensure_root(origin)?;
             Self::do_dissolve_network(netuid)
         }
+
+        /// --- Claims the root emissions for a coldkey.
+        /// # Args:
+        /// * 'origin': (<T as frame_system::Config>Origin):
+        /// 	- The signature of the caller's coldkey.
+        ///
+        /// # Event:
+        /// * RootClaimed;
+        /// 	- On the successfully claiming the root emissions for a coldkey.
+        ///
+        /// # Raises:
+        ///
+        #[pallet::call_index(121)]
+        #[pallet::weight((
+            Weight::from_parts(117_000_000, 7767)
+                .saturating_add(T::DbWeight::get().reads(11_u64))
+                .saturating_add(T::DbWeight::get().writes(4_u64)),
+            DispatchClass::Normal,
+            Pays::Yes
+        ))]
+        pub fn claim_root(origin: OriginFor<T>) -> DispatchResultWithPostInfo {
+            let coldkey: T::AccountId = ensure_signed(origin)?;
+
+            Self::maybe_add_coldkey_index(&coldkey);
+
+            let weight = Self::do_root_claim(coldkey);
+            Ok((Some(weight), Pays::Yes).into())
+        }
+
+        /// --- Sets the root claim type for the coldkey.
+        /// # Args:
+        /// * 'origin': (<T as frame_system::Config>Origin):
+        /// 	- The signature of the caller's coldkey.
+        ///
+        /// # Event:
+        /// * RootClaimTypeSet;
+        /// 	- On the successfully setting the root claim type for the coldkey.
+        ///
+        #[pallet::call_index(122)]
+        #[pallet::weight((
+            Weight::from_parts(6_000_000, 0).saturating_add(T::DbWeight::get().writes(1_u64)),
+            DispatchClass::Normal,
+            Pays::Yes
+        ))]
+        pub fn set_root_claim_type(
+            origin: OriginFor<T>,
+            new_root_claim_type: RootClaimTypeEnum,
+        ) -> DispatchResult {
+            let coldkey: T::AccountId = ensure_signed(origin)?;
+
+            Self::maybe_add_coldkey_index(&coldkey);
+
+            Self::change_root_claim_type(&coldkey, new_root_claim_type);
+            Ok(())
+        }
+
+        /// --- Sets the root claim type for the coldkey.
```

Comment:
this description doesn't match the function (same for both function descriptions below)